Sources/Purchasing/Purchases.swift

```diff
@@ -1828,13 +1862,17 @@ public extension Purchases {
                                                       observerMode: Bool,
                                                       userDefaults: UserDefaults?,
                                                       storeKit2Setting: StoreKit2Setting,
+                                                      storeKitTimeoutSeconds: Int,
+                                                      networkTimeoutSeconds: Int,
                                                       dangerousSettings: DangerousSettings?) -> Purchases {
         let purchases = Purchases(apiKey: apiKey,
```

Comment:
Should we mark the other configure methods as deprecated?

Reply:
Then you could move them to `Deprecations.swift` to clean up this file of all those overloads :D

Reply:
✅